src/main/scala/com/mwz/sonar/scala/qualityprofiles/RecommendedQualityProfile.scala

```diff
@@ -76,55 +53,30 @@ final class RecommendedQualityProfile extends BuiltInQualityProfilesDefinition {
 
 private[qualityprofiles] object RecommendedQualityProfile {
   final val ProfileName: String = "Recommended by sonar-scala"
-  final val Blacklist: Set[String] = Set(
-    // Scalastyle
-    "block.import", // avoid block imports
-    "lowercase.pattern.match", // lowercase pattern match
-    "no.newline.at.eof", // no newline at EOF
-    "pattern.match.align", // pattern match align
-    "underscore.import", // avoid wildcard imports
-    // Scapegoat
-    "com.sksamuel.scapegoat.inspections.naming.ClassNames", // exists in Scalastyle (class.name)
-    "com.sksamuel.scapegoat.inspections.string.EmptyInterpolatedString", // exists in Scalastyle (empty.interpolated.strings)
-    "com.sksamuel.scapegoat.inspections.unneccesary.UnnecessaryReturnUse" // exists in Scalastyle (return)
-  )
-  final val Severities: Map[String, Severity] = Map(
-    "covariant.equals" -> Severity.MAJOR,
-    "equals.hash.code" -> Severity.MAJOR,
-    "illegal.imports" -> Severity.MAJOR,
-    "null" -> Severity.MAJOR,
-    "var.field" -> Severity.MAJOR,
-    "var.local" -> Severity.MAJOR,
+  final val RuleOverrides: Overrides = Overrides(
```

Comment:
What about having two `Overrides` ?
One for **Scalastyle** and the other for **Scapegoat**

Reply:
yeah sure, we can split it up